tests/integration/nupic/opf/hotgym_regression_test.py

```diff
@@ -61,10 +60,10 @@ def testHotgymRegression(self):
                          "steps=1:window=1000:field=consumption")
         lastRow = collections.deque(reader, 1)[0]
 
-      # Changes that affect prediction results will cause this test to fail.
-      # If the change is understood and reviewers agree that there has not been a
+      # Changes that affect prediction results will cause this test to fail. If
+      # the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
-      self.assertAlmostEqual(float(lastRow[14]), 5.90725098207)
+      self.assertLess(abs(float(lastRow[14])-5.90725098207), 0.04)
```

Comment:
You can instead add the parameter `delta=0.04` to the `assertAlmostEqual` method.